src/snapred/backend/dao/state/FocusGroupParameters.py

```diff
@@ -1,3 +1,5 @@
+# TODO delete
```

Comment:
Should this be deleted as part of this pr?

Reply:
It had created some error last time I tried to delete it.  I'll try again.